### DISCUSSION TITLE
Newbie here and need a bit of advice

### DISCUSSION THREAD

**Discussion by MIDIculous** [ant-media/Ant-Media-Server#5544] [Q&A]
I'm new to Ant Media and to give things a try I was able to install the app via Linode. I do understand I can access my server using the iPAddress:5080, but this is not secure. How do I make this secure?

**Comment by yashtandon113** [comment]
@MIDIculous 
Thanks for your interest in Ant Media Server.
In order to secure your server, you need to enable the SSL certificate on the server. Please check out the following [SSL document](https://antmedia.io/docs/guides/installing-on-linux/setting-up-ssl/).

Their are multiple ways to get the SSL certificate. I just want to clarify that, you can get the free domain only when using Enterprise Edition of AMS.

If you are using Community Edition of the server, you can use your own domain.

Please let us know if you have any questions.

**Reply by MIDIculous** [reply]
Ok, so the the SSL doesn't have to come from Linode? I assumed that the SSL would have to come from Linode and I would have to use my custom domain there. 

Sorry for the newbie questions as I'm coming from the cPanel/WHM world where everything is self-hosted. This hybrid hosting method is new to me.
